utils/chain-spec-builder/src/main.rs

```diff
@@ -241,6 +264,12 @@ fn genesis_constructor(
         )
     };
 
```

Comment:
What do you think about functional style or you prefer the imperative style more?
let initial_account_balances = initial_balances_path.map(|path| {initial_balances::from_json(path.as_path())}).unwrap_or_else(vec![]);